resources/views/volumes/clone.blade.php

```diff
@@ -93,83 +95,76 @@ class="text-muted">#<span v-text="file.id"></span></span> <span
                     </div>
                     <div>
                         <div class="checkbox">
-                            <div v-cloak>
+                            <div>
                                 <label><input type="checkbox" class="checkbox" id="fileLabels"
-                                              v-model="cloneFileLabels" name="clone_file_labels" value="1">
-                                    <span v-text="'Clone '+getFileType(false)+' labels'"></span>
-                                    <div class="form-group{{ $errors->has('clone_file_labels') ? ' has-error' : '' }}">
+                                              v-model="filterFileLabels" name="clone_file_labels" value="1">
+                                    <span>Clone file labels</span>
+                                    <div class="form-group{{ $errors->has('clone_file_labels') ? ' has-error' : '' }}" v-cloak>
```

Comment:
No `form-group` required for a checkbox. The `has-error` could be added to the the `<div>` at line 96.

Also there is no `v-if` here, so no `v-cloak` is required.

Also any empty `<div>` in the template (e.g. line 98) is unnecessary.